tests/drivers/retained_mem/api/testcase.yaml

```diff
@@ -17,3 +17,10 @@ tests:
     tags:
       - drivers
       - retained_mem
+  drivers.retained_mem.api.ram.nrf54l15pdk_nrf54l15_cpuapp:
+    platform_allow: nrf54l15pdk_nrf54l15_cpuapp
+    extra_args:
+      - DTC_OVERLAY_FILE="nrf54l15pdk_nrf54l15_cpuapp_ram.overlay"
```

Comment:
why not remove `_ram` suffix, remove this  configuration and just add `nrf54l15pdk_nrf54l15_cpuapp` to `platform_allow` for `drivers.retained_mem.api.ram` case.

Reply:
fixed